migrations/db/migrations/20221207154255_create_pgsodium_and_vault.sql

```diff
@@ -6,6 +6,10 @@ grant pgsodium_keyiduser to postgres with admin option;
 grant pgsodium_keyholder to postgres with admin option;
 grant pgsodium_keymaker  to postgres with admin option;
 
+grant execute on function pgsodium.crypto_aead_det_decrypt(bytea, bytea, uuid, bytea) to service_role;
+grant execute on function pgsodium.crypto_aead_det_encrypt(bytea, bytea, uuid, bytea) to service_role;
+grant execute on function pgsodium.crypto_aead_det_keygen to service_role;
```

Comment:
I have no idea why, but `pgsodium/after-create.sql` are not run during `create extension pgsodium` above, but it's run if I recreate the extension (tested on staging). This was the cause of the build failures.

I'll just add these since the grants are supposed to be idempotent.